src/Routes.js

```diff
@@ -4,7 +4,8 @@ import React from 'react';
 import some from 'lodash/some';
 
 import Insights from './Routes/Insights';
-import CloudManagement from './Routes/CloudManagement';
+import Cost from './Routes/Cost';
+#import CloudManagement from './Routes/CloudManagement';
```

Comment:
Another one here. If you're trying to comment it out, you'll need to do
`// import CloudManagement from './Routes/CloudManagement'; `